mps_youtube/config.py

```diff
@@ -225,6 +225,38 @@ def check_win_size(size):
     else:
         return dict(valid=True, value=size)
 
+def check_being_dict(user_dict):
+    """ check if user string is correct dict, set it to global options,
+        since pafy accepts ydl_opts dict as ctor argument. """
+    try:
+        import ast
+    except ImportError:
+        m = "Your Python is too old. Use at least 2.4 to use this option."
```

Comment:
Mysyt no longer supports (nor functions in) any version of python 2, so this test should be unnecessary.